azure/vm.sh

```diff
@@ -136,3 +136,10 @@ fn azure_vm_delete(name, group) {
 			--resource-group $group
 	)
 }
+
+fn azure_vm_get_ip_address(name, group, iface_index, ip_index) {
+        info <= azure vm list-ip-address $group --json
+        #echo $ips
+        ip <= echo $info | jq -r ".[0].networkProfile.networkInterfaces[" + $iface_index + "].expanded.ipConfigurations[" + $ip_index + "].privateIPAddress"
```

Comment:
auheuaheuaeau pretty simple right ? @tiago4orion feel the love for some maps + JSON support here :heart: